src/services/Type1GasEstimator.ts

```diff
@@ -2,7 +2,7 @@ import { EstimateParams, GasEstimation } from '../types/GasEstimation';
 
 export class Type1GasEstimator {
   public static estimate(params: EstimateParams): GasEstimation {
-    const baseFee = params.prices.length > 2 ? Type1GasEstimator.calculateBaseGasPrice(params) : 0;
+    const baseFee = params.prices.length > 2 ? Type1GasEstimator.calculateBaseGasPrice(params) : params.currentGasPrice;
```

Comment:
Arbitrum blocks contains only 1 transaction, so we were replicating a gas price of `0` always.

Reply:
Could params.currentGasPrice be null or undefined? 
Not sure if we should add a condition to avoid unexpected value.

Reply:
i guess it may if provider.getGasPrice() fails

Reply:
Nope. It's always a BigNumber
https://docs.ethers.io/v5/api/providers/provider/#Provider-getGasPrice

Reply:
i guess it will throw in failure, otherwise always return a bn